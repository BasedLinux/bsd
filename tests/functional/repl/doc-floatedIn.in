:l doc-comments.bsd
:doc floatedIn
