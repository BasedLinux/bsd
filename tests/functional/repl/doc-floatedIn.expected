Bsd <bsd version>
Type :? for help.

bsd-repl> :l doc-comments.nix
Added <number omitted> variables.

bsd-repl> :doc floatedIn
Function `floatedIn`\
  â€¦ defined at /path/to/tests/functional/repl/doc-comments.nix:21:5


This also works.
